realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -466,30 +522,16 @@ public void onSuccess() {
                             .equalTo("id", changeRequestId)
                             .findFirstAsync();
 
+
                     // Wait for it to be processed
-                    managedChangeRequest.addChangeListener(new RealmChangeListener<PermissionChange>() {
+                    RealmObject.addChangeListener(managedChangeRequest, new RealmChangeListener<PermissionChange>() {
```

Comment:
I think this would be easier to read if there were a newline before the `new`.  I believe that our style guidelines support that...